shared/constants/types/crypto.tsx

```diff
@@ -18,13 +18,22 @@ type DecryptTab = 'decryptTab'
 type SignTab = 'signTab'
 type VerifyTab = 'verifyTab'
 
+type EncryptOutput = 'encryptOutput'
```

Comment:
likely don't need this, can just use typeof the constants likely

Reply:
Was attempting to avoid cyclical dependencies